docker/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM golang:1.14
-ENV GOBIN /go/bin
-RUN go get github.com/stellar/stellar-etl
```

Comment:
Running the original Dockerfile gave me build errors. I think these changes is what we want anyways though. It follows the pattern of other Go Dockerfiles we have